packages/nocodb/src/lib/models/Audit.ts

```diff
@@ -44,6 +44,9 @@ export default class Audit implements AuditType {
       forceAwait: process.env['TEST'] === 'true',
     }
   ) {
+    if (process.env['NC_DISABLE_AUDIT'] ?? false) {
+      return;
+    }
```

Comment:
Since here you are just checking env value is trruthy do the same in appInfo API as well.

`process.env.NC_DISABLE_AUDIT === 'true' ? false : true,` => `!process.env.NC_DISABLE_AUDIT,`

Reply:
`process.env.NC_DISABLE_AUDIT` is a string. `!"false" -> false` and `!"true" -> false`.